debug_toolbar/static/debug_toolbar/css/toolbar.css

```diff
@@ -34,6 +34,13 @@
     --djdt-button-border-color: var(--djdt-table-border-color);
     --djdt-pre-border-color: var(--djdt-table-border-color);
     --djdt-raw-border-color: var(--djdt-table-border-color);
+
+    --djdt-health-bg-1: #4b3f1b;
+    --djdt-health-color-1: #ffe761;
+    --djdt-health-bc-1: #ffcc00;
```

Comment:
What is `bc`? Background color? I think we prefer using variable names without abbreviations.